### ISSUE TITLE
Work assignments

### ISSUE THREAD

**Issue by cozyycoding** [JakeCode121/Washington_Trail#28]
@JakeCode121  - do a new CSS for the index page
@dsl28  - work with CSS border for text pages
@mld137  - proofread XML for the journal and fixing special characters

**Comment by JakeCode121** [comment]
@Bennediction @haggis78 @mld137 @dsl28 

Hello,

I cleaned up the web page index, and I wanted to put titles under the commanders.  However, after hours of messing with it, it give up.

I added the French Commander and added the French flag, as well as changed the cover.

Have a good night.

**Comment by JakeCode121** [comment]
Hello,

I fixed the parchment issue on the website and was able to fix the index pages with all of the information.
@mld137 @dsl28 Remember to go in and add your personal information on the process page.  Lets do a quick 10-15 min huddle on Monday to discuss who is going to talk about what.

@Bennediction @haggis78 I believe my internet went out: I apologize.  I kept messing with and it seems good to go. Thank you for all your help, and enjoy you holiday. @JakeCode121
